src/loop_in_chunks.cpp

```diff
@@ -508,6 +524,8 @@ void fields::loop_in_chunks(field_chunkloop chunkloop, void *chunkloop_data, con
       }
     } while (ishift != min_ishift);
   }
+  int vol_sum_all = sum_to_all(vol_sum);
+  if (use_symmetry && vol_sum_all != original_vol) master_printf("WARNING vol mismatch:, original_vol %i, looped vol_sum %i \n", original_vol, vol_sum_all);
```

Comment:
When ``use_symmetry=false``,  e.g. 
https://github.com/NanoComp/meep/blob/75ced5296a91d8e49ad61b1bb07093e7abbb95e9/src/sources.cpp#L460
the symmetry operations won't be applied to chunks and thus don't intersect the entire original volume.